inst/shiny/modules/tab_nca.R

```diff
@@ -139,13 +139,20 @@ tab_nca_server <- function(id, adnca_data, grouping_vars) {
                 slope_rules$slopes_groups(),
                 check_reasons = TRUE
               ) %>%
+              # Perform PKNCA parameter calculations
               PKNCA_calculate_nca() %>%
               # Add bioavailability results if requested
               add_f_to_pknca_results(settings()$bioavailability) %>%
               # Apply standard CDISC names
               mutate(
                 PPTESTCD = translate_terms(PPTESTCD, "PKNCA", "PPTESTCD")
               ) %>%
+              # Apply flag rules to mark results in the `exclude` column
+              PKNCA_hl_rules_exclusion(
+                rules = isolate(settings()$flags) |>
+                  purrr::keep(~ .x$is.checked) |>
+                  purrr::map(~ .x$threshold)
```

Comment:
nitpick, style: There is a general shift away from using formulas in R and instead going for anonymous functions, since they are a) more verbose about arguments without much more boilerplate, b) behaviour is more consistent and predictable and c) they are supported in various forms across many languages, so full-stack developers who are not experts in R have an easier time understanding the syntax.

```suggestion
                  purrr::keep(\(x) x$is.checked) |>
                  purrr::map(\(x) x$threshold)
```